scripts/submit-work-service.py

```diff
@@ -132,7 +132,9 @@ def build_jobs(dictionary, batch_client, job_queue, job_def, work_bucket, datast
         tile_index = str(key[2])
         tile_id = str(((key[2]) << 3) | key[1])
         job_name = str(key[0]) + '_' + str(tile_id)
-        files = ','.join(val)
+        # just use the shared prefixes of the files up to the last directory
+        files = set([ f[:f.rfind('/') + 1] for f in val ])
+        files = ','.join(files)
```

Comment:
so this finds the last occurrence of `/` and truncates the string to that, and throws it in a `set` so we get all the unique prefixes